color_windows.go

```diff
@@ -11,8 +11,10 @@ func init() {
 	// https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#output-sequences
 	var outMode uint32
 	out := windows.Handle(os.Stdout.Fd())
-	if err := windows.GetConsoleMode(out, &outMode); err == nil {
-		outMode |= windows.ENABLE_PROCESSED_OUTPUT | windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING
-		_ = windows.SetConsoleMode(out, outMode)
+	if err := windows.GetConsoleMode(out, &outMode); err == nil || NoColor {
```

Comment:
I wrote more here, but let's remove `NoColor` integration. It wouldn't work during runtime. See my comment: https://github.com/fatih/color/pull/186#discussion_r1121285897

Also we want to return if GetConsoleMode doesn't work, so err should be non-nil

```suggestion
	if err := windows.GetConsoleMode(out, &outMode); err != nil {
```

Reply:
> Also we want to return if GetConsoleMode doesn't work, so err should be non-nil.

D'oh. Fixed.

So, should we just abandon this PR then, since it just attempts the NoColor integration?